_posts/2016/2016-03-17-team-orange-mallorca.md

```diff
@@ -2,7 +2,6 @@
 layout: post
 title: "Productive team offsite on Mallorca"
 date: "2016-03-17 16:03:17"
-icon: suitcase
 categories: working-at-epages events
 authors: ["Alejandra, Benjamin and Heiko"]
 ---
```

Comment:
`image: ....` is missing here. 

Offtopic: 

Wouldn't it make sense to derive the image path from the given category instead of hardcoding an image path into the YAML frontmatter? ...as this seems to be very often related in this redesign.

Hardcoded duplicate paths are not very maintainable in general. A lot of jekyll blogs use just one category to make this correlation work. If there is a need for more tech terms they add them to another `tags: ...` field. 

Maybe this might be worth to consider. ;-)